### PR TITLE
Removal of transformation scripts

### PR DESCRIPTION
**Pull request by mereolog**
This pull request removes the transformation scripts from the public repository.

### CODE REVIEW

**Comment by aliariff** [comment]
@mereolog and what is the plan here? do we move it somewhere else? or these transformations are not required anymore?

**Comment by mereolog** [comment]
> @mereolog and what is the plan here? do we move it somewhere else? or these transformations are not required anymore?

This PR is done at the request of the PA.
I moved the scripts to https://github.com/edmcouncil/idmp-transformations - please check whether I moved them all.

File: .github/workflows/unit_tests_run.yml
Comment (aliariff) [inline .github/workflows/unit_tests_run.yml:1]: we can also remove this part
https://github.com/edmcouncil/idmp/pull/641/files#diff-f52a0b5af97cf0553b8b172795c6cc3720e033c78b3a5adecf9bd347c098a57cR9-R16

```
 workflow_dispatch:
    inputs:
      gsrs_version:
        description: "GSRS dataset version"
        required: false
      gsrs_cache_rebuild:
        description: "Whether to rebuild cache"
        required: false
```

**Review by aliariff — APPROVED** [review]

**Review by Heiner-Oberkampf — APPROVED** [review]
Agreed with transition to non-public as decided by PA

**Pull request merged**